chart/base/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
@seansund 
Do we need to add this to the repo?

Reply:
I removed the file and added it to gitignore